SignalServiceKit/src/Messages/Interactions/OWSContact.m

```diff
@@ -58,7 +58,7 @@ - (BOOL)ows_isValid
     return YES;
 }
 
-- (NSString *)localizedLabel
+- (nullable NSString *)localizedLabel
 {
     switch (self.phoneType) {
         case OWSContactPhoneType_Home:
```

Comment:
Wait, is this necessary?  We're doing something like this in each implementation so that we fallback (like iOS) to a label like "phone" for phone numbers, etc.

```
   if (self.label.ows_stripped.length < 1) {
                 return NSLocalizedString(@"CONTACT_PHONE", @"Label for a contact's phone number.");
             }
```

Reply:
I don't understand your question - this was *always* nullable, I didn't change that. Now it's just annotated correctly.

Reply:
Ohhhhhh. No, you're right. Sheesh. I'll revert this part.

Reply:
Okay, but does this method even need to be nullable?